easybuild/easyconfigs/r/Rgurobi/Rgurobi-9.5.0-foss-2021b-R-4.1.2.eb

```diff
@@ -0,0 +1,43 @@
+##
+# This is a contribution from SIB Swiss Institute of Bioinformatics
+# Homepage:     https://www.sib.swiss/research-infrastructure/competence-centers/vital-it
+#
+# Authors::     Sebastien Moretti <sebastien.moretti@sib.swiss>
+#
+##
+
+easyblock = 'Binary'
+
+name = 'Rgurobi'
+version = '9.5.0'
+versionsuffix = '-R-%(rver)s'
```

Comment:
@boegel I've seen in some other recent easyconfigs that this specific R department has been removed. Just wondering whether that should apply here?